FWCore/Integration/test/BuildFile.xml

```diff
@@ -77,6 +77,11 @@
     <use name="FWCore/Utilities"/>
   </bin>
 
+  <bin file="TestIntegration.cpp" name="TestIntegrationProcessBlock">
+    <flags TEST_RUNNER_ARGS=" /bin/bash FWCore/Integration/test run_TestProcessBlock.sh"/>
+    <use name="FWCore/Utilities"/>
+  </bin>
+
```

Comment:
Commenting on my own PR. I am working on changing this to the new way of running unit tests with shell scripts.

@smuzaffar Does the PRE_TEST flag in BuildFile.xml force the order of tests? Does it require that the tests identified with that flag run and complete before the test runs? If I use PRE_TEST, then I can probably break run_TestProcessBlock.sh into several pieces to get some more concurrency. Can I have multiple PRE_TEST flags for the same test? (I looked in the SWGuide TWIKI, https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideBuildFile, but didn't see any description of how this all works.)

Reply:
yes PRE_TEST force the order and will start the test after PRE_TEST(s) are done. Yes, you can have multiple pre tests e.g.
```
<flags PRE_TEST="A B C"/>
or
<flags PRE_TEST="A"/>
<flags PRE_TEST="B"/>
<flags PRE_TEST="C"/>
```
both sould work.